test/TEST-01-BASIC/test.sh

```diff
@@ -53,7 +53,7 @@ Description=Testsuite service
 After=multi-user.target
 
 [Service]
-ExecStart=/bin/bash -c 'set -x; systemctl --failed --no-legend --no-pager > /failed ; echo OK > /testok; while : ;do echo "testsuite service waiting for journal to move to /var/log/journal" > /dev/console ; for i in /var/log/journal/*;do [ -d "\$i" ] && echo "\$i" && break 2; done; sleep 1; done; sleep 1; exit 0;'
+ExecStart=/bin/bash -c 'set -x; systemctl --failed --no-legend --no-pager > /failed ; echo OK > /testok;'
```

Comment:
@filbranden , fixed.
Please, take a look: https://github.com/evverx/systemd/commit/7d97774cbb4868a6966e9b8b604457f4efeb63a2. I like `/bin/bash -xc` but use `/bin/sh -x -c`. It's consistent with your style in [`test-execute`](https://github.com/systemd/systemd/pull/1819/files)